pennylane_cirq/simulator_device.py

```diff
@@ -71,225 +59,115 @@ class SimulatorDevice(CirqDevice):
     short_name = "cirq.simulator"
 
     def __init__(self, wires, shots=1000, analytic=True, qubits=None):
-        super().__init__(wires, shots, qubits)
+        super().__init__(wires, shots, analytic, qubits)
 
-        self.initial_state = None
-        self.simulator = cirq.Simulator()
-        self.result = None
-        self.measurements = None
-        self.state = None
-        self.analytic = analytic
```

Comment:
Here by removing this attribute, it will be created implicitly, right? (Just checking if this was intended).

Reply:
You mean `analytic`? It is now created in the superclass.

Reply:
Yep! Cool!